pages/search/index.vue

```diff
@@ -12,6 +12,9 @@
           :is-loading="isLoading"
           @submit:searchForm="submitSearchForm"
         />
+        <SearchSelectedFacets
+          :selected="typeFacet"
```

Comment:
pass the whole facets in and get the kind of facet from the key, then maybe create a sub component.